shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -325,10 +329,12 @@ public MotionEvent toMotionEvent(
             .toArray(new PointerCoords[touch.pointerCount]);
 
     if (!usingVirtualDiplays && trackedEvent != null) {
+      // TODO (kaushikiska): investigate why we can not use the tracked
```

Comment:
I think the reason is that the action was mutated while it was waiting for the reply from the framework